packages/fiori/src/ShellBarBrandingTemplate.tsx

```diff
@@ -0,0 +1,23 @@
+import type ShellBarBranding from "./ShellBarBranding.js";
+
+export default function ShellBarBrandingTemplate(this: ShellBarBranding) {
+	return (
+	  <a
+			part="root"
```

Comment:
This css part seems not be be used anywhere internaly nor documented as public. I think we don't need it currently. If there is hardcoded CSS on the anchor, we can move it on the host and inherit to the anchor.